app/src/org/commcare/android/logging/ForceCloseLogSerializer.java

```diff
@@ -65,6 +65,7 @@ protected void serializeLog(LogEntry entry) throws IOException {
                     forceCloseEntry.getMessage(), serializer);
             AndroidLogSerializer.writeText("app_build",
                     forceCloseEntry.getAppBuildNumber() + "", serializer);
+            // TODO: android_version and device_model are part of the report header now, so might no longer be needed here
```

Comment:
can we remove these right away ? Or backlog a ticket with due date so that we don't forget.

Reply:
@shubham1g5 this will require a migration, so I created a ticket and will advocate to be added to the next DEET Week.

Reply:
Think I am fine having them in models, but we should just remove writes to them here to make it clear that they are being read from header and not from here.